ResourceStatusSystem/Utilities/InfoGetter.py

```diff
@@ -1,217 +1,57 @@
-# $HeadURL: $
-''' InfoGetter
+""" InfoGetter
 
   Module used to map the policies with the CS.
 
-'''
+"""
 
 import copy
 
-from DIRAC                                import S_OK, S_ERROR
+from DIRAC                                import S_OK, S_ERROR, gConfig, gLogger
 from DIRAC.ResourceStatusSystem.Utilities import RssConfiguration, Utils
 
-from DIRAC                                       import gConfig, gLogger
-
-
-
 __RCSID__ = '$Id: $'
 
-class InfoGetter:
-  """
-    Class InfoGetter is in charge of getting information from the RSS Configurations
+
+def sanitizedecisionParams( decisionParams ):
```

Comment:
To be honest I think the whole class is useless here. Just functions would be enough